h2o-algos/src/main/java/hex/tree/drf/DRFModel.java

```diff
@@ -73,4 +77,17 @@ public DRFParameters() {
     }
   }
 
+  @Override
+  public Model<DRFModel, DRFModel.DRFParameters, DRFModel.DRFOutput>.RawDataStreamWriter getRawDataStream() {
+    return new RawDataStreamWriter();
+  }
+
+  public class RawDataStreamWriter
+          extends SharedTreeModel<DRFModel, DRFModel.DRFParameters, DRFModel.DRFOutput>.RawDataStreamWriter {
+    @Override
+    protected void writeExtraModelInfo() throws IOException {
+      super.writeExtraModelInfo();
```

Comment:
We need to document this pattern (it is error-prone), or you can just use strategy pattern.